package.json

```diff
@@ -126,7 +126,7 @@
     "lint": "eslint .",
     "dev": "cross-env BABEL_ENV=default webpack --watch",
     "test": "npm run lint && npm run test-unit",
-    "ci": "concurrently \"npm run lint && npm run build\" \"npm run test-unit:coverage-ci\"",
+    "ci": "concurrently \"npm run build\" \"npm run test-unit:coverage-ci\"",
```

Comment:
Do we need to execute `npm run build` here, too? It's executed as part of `bin/run-wp-unit-tests.sh` for every PHP version.

Reply:
It was added later to the PHP unit tests to build the parser. If the parser is no longer built this way in the future (say a Pure PHP parser), this could be removed.

While keeping it here ensure the build is tested as part of the JS unit tests which is more logical to me.

Reply:
We moved lint out, so we could do the same with the build, and call `npm run test-unit:coverage-ci` directly from Travis. We can leave it as it is, too. I don't think it makes any difference in the end.